react-app/src/components/projects/AddProject.tsx

```diff
@@ -221,6 +230,14 @@ const AddProject = () => {
               />
             </Grid>
           ))}
+          <Grid item xs={12}>
+            <TextFormField
+              disabled={watch.find((a) => a.TaskId === exemptionTask.Id)?.IsCompleted != true}
+              fullWidth
+              name={'ExemptionNote'}
+              label={'Exemption rationale note'}
+            />
+          </Grid>
```

Comment:
In old PIMS, this was a required field when there was a request for exemption. 
Also suggest making it a multiline input and hiding when not required.

```suggestion
          {watch.find((a) => a.TaskId === exemptionTask.Id)?.IsCompleted === true ? (
            <Grid item xs={12}>
              <TextFormField
                fullWidth
                name={'ExemptionNote'}
                label={'Exemption rationale note'}
                required
                multiline
                minRows={2}
                rules={{
                  required: 'Rationale required when requesting exemption.',
                }}
              />
            </Grid>
          ) : (
            <></>
          )}
```

Reply:
I think some web design purists may argue that hiding fields this way is bad practice but I personally think it's alright here. Implemented these changes.